lit_llama/model.py

```diff
@@ -220,7 +220,7 @@ def forward(
 
         # causal self-attention; Self-attend: (B, nh, T, hs) x (B, nh, hs, T) -> (B, nh, T, T)
         #  att = (q @ k.transpose(-2, -1)) * (1.0 / math.sqrt(k.size(-1)))
-        #  att = att.masked_fill(self.bias[:,:,:T,:T] == 0, float('-inf'))
+        #  att = att.masked_fill(mask[:,:,:T,:T] == 0, float('-inf'))
```

Comment:
We used to have it here: https://github.com/Lightning-AI/lit-llama/commit/1598d127b9a30684180ac0b5fa1d872e8fabe513#diff-e076d0744d44408d4cd6e67de57a60beb2658f85405963b69f888b2b5c413092R97-R98

But mask was added recently. LGTM!